src/data-mgt/python/events-notifs/utils.py

```diff
@@ -0,0 +1,155 @@
+import traceback
+
+import requests
+import urllib3
+from firebase_admin import messaging
+
+from config import configuration
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def compose_notification_message(pm2_5, site_name):
+    if 0.00 <= pm2_5 <= 12.09:
```

Comment:
Thanks @NoahNsimbe 
How are we customising (personalising) these notifs for different app users?